normandy/recipes/api/serializers.py

```diff
@@ -37,6 +38,25 @@ class Meta:
         ]
 
 
-class BundleSerializer(serializers.Serializer):
+class BundleSerializer(serializers.ModelSerializer):
     recipes = RecipeSerializer(many=True)
     country = serializers.CharField()
+
+
+class RecipeVersionSerializer(serializers.ModelSerializer):
+    date_created = serializers.SerializerMethodField(read_only=True)
+    recipe = serializers.SerializerMethodField(read_only=True)
```

Comment:
ah! i was trying to figure out how to do that. i was trying `revision__date_created` etc. with no luck. thanks!